common/src/main/kotlin/org/neo4j/connectors/kafka/data/PropertyType.kt

```diff
@@ -0,0 +1,366 @@
+/*
+ * Copyright (c) "Neo4j"
+ * Neo4j Sweden AB [https://neo4j.com]
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.neo4j.connectors.kafka.data
+
+import java.nio.ByteBuffer
+import java.time.LocalDate
+import java.time.LocalDateTime
+import java.time.LocalTime
+import java.time.OffsetDateTime
+import java.time.OffsetTime
+import java.time.ZoneOffset
+import java.time.ZonedDateTime
+import java.time.format.DateTimeFormatter
+import java.time.temporal.TemporalQueries
+import kotlin.reflect.KClass
+import org.apache.kafka.connect.data.Schema
+import org.apache.kafka.connect.data.SchemaBuilder
+import org.apache.kafka.connect.data.Struct
+import org.neo4j.driver.Values
+import org.neo4j.driver.types.IsoDuration
+import org.neo4j.driver.types.Point
+
+@Suppress("UNCHECKED_CAST")
+object PropertyType {
+  internal const val MONTHS = "months"
+  internal const val DAYS = "days"
+  internal const val SECONDS = "seconds"
+  internal const val NANOS = "nanoseconds"
+  internal const val SR_ID = "srid"
+  internal const val X = "x"
+  internal const val Y = "y"
+  internal const val Z = "z"
+  internal const val DIMENSION = "dimension"
+  internal const val TWO_D: Byte = 2
+  internal const val THREE_D: Byte = 3
+
+  internal const val BOOLEAN = "B"
```

Comment:
would it be too wasteful to use longer values?

Reply:
I wasn't sure too, probably for JSON yes - others most probably not. What are your thoughts on this?

Reply:
I find it a bit too terse, but let's keep it like this for now